text/code-standards/formatting.md

```diff
@@ -0,0 +1,20 @@
+# Formatting
+
+All Rust projects should follow the same formatting standard. This standard is
+to be documented in a `rustfmt.toml` file in the top level of each project.
+See below for an example of the default values for the file.
+
+```toml
+fn_args_layout = "Tall"
+hard_tabs = false
+match_arm_leading_pipes = "Never"
+max_width = 100
+merge_derives = true
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
```

Comment:
This auto formats imports into a nice format for diffing (requires nightly fmt though):

```suggestion
imports_layout = "Vertical"
imports_granularity = "Crate"
tab_spaces = 4
```


```
use foo::{
    a,
    b
}
```